packages/sdk-drop-in/tests/iframe.test.ts

```diff
@@ -0,0 +1,45 @@
+import { test } from '@playwright/test';
+
+const getMockConfig = () => ({
+  client_id: '2afafd88-7b8c-4ed2-b0f0-652c0db600b7',
+  client_secret: 'ae6b5ece-51a1-4fcf-97d2-d10bb0de0612',
+  entity_user_id: '0771f748-2e98-400e-adb4-20ebefd77f5f',
+  grant_type: 'entity_user',
+});
+
+const consumerPage = '/monite-iframe-app-consumer/receivables';
+const authTokenPath = '/v1/auth/token';
+
+test.describe('Monite Iframe Integration', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(consumerPage);
+  });
+
+  test('should fetch token and render iframe', async ({ page }) => {
+    const [] = await Promise.all([
+      page.waitForRequest((request) => request.url().includes(authTokenPath)),
+      page.waitForSelector('iframe.monite-iframe-app'),
+    ]);
+
+    const tokenResponse = getMockConfig();
+
+    await page.route(`https://api.dev.monite.com/${authTokenPath}`, (route) => {
+      route.fulfill({
+        contentType: 'application/json',
+        body: JSON.stringify(tokenResponse),
+      });
+    });
+  });
+
+  test('should see the sales and navigate inside the iframe', async ({
```

Comment:
As part of iframe performance tests, do you think it's necessary to add user scenarios? I thought it was enough to test the basic functionality.

Reply:
My idea is to check if routing inside the iframe actually works and some basic actions can be performed. Additionally, by including user scenarios, we can ensure comprehensive coverage and validate performance under realistic conditions

Reply:
Thank you for sharing these insights. I was unaware that we already have test coverage within the SDK-React. In that case, it is indeed redundant